apis/externalsecrets/v1alpha1/secretstore_vault_types.go

```diff
@@ -173,6 +171,21 @@ type VaultKubernetesAuth struct {
 	// +optional
 	ServiceAccountRef *esmeta.ServiceAccountSelector `json:"serviceAccountRef,omitempty"`
 
+	// Optional audiences field that will be used to request a Kubernetes projected service
```

Comment:
Please advise if it is okay to have this in the `kubernetes` section or if you would prefer to have this in the `jwt` section or in completely new section (`kubernetesProjectedServiceAccountToken`?)

Reply:
I think it would make sense to be in the `jwt` section, or maybe a new section. @moolen any thoughts on this?

Reply:
Moved it the `jwt` section`.